src/utils/RouteUtils.js

```diff
@@ -15,8 +14,14 @@ import ParseRouteUtils from './ParseRouteUtils';
  * @param arriveBy
  * @returns {*}
  */
-/* eslint-disable no-param-reassign */
-async function createFinalRoute(routeBus, routeWalking, start, end, departureTimeQuery, arriveBy: boolean) {
+async function createFinalRoute(
+    routeBus: Array<Object>,
+    routeWalking: Object,
+    start: string,
+    end: string,
+    departureTimeQuery: number,
+    arriveBy: boolean,
```

Comment:
what is arriveBy and why is it a boolean? is it like if the user arrived or not? cuz arriveby sounds like it should be a time

Reply:
difficult to change this because it's in production with this name already on the client-side, changing it will make it more difficult to debug errors